install/kubernetes/helm/istio/charts/security/templates/cleanup/ClusterRole.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
@zxDiscovery @morvencao can you help check if we can the folder `cleanup` under `templates`? Does this works? I found most of the charts only including files under `templates`, but no folder.

Reply:
@gyliu513 Local tests show we can put some templates into a folder like `cleanup`.

Reply:
Good to know, thanks @morvencao